### DISCUSSION TITLE
Runninn long experiment in breaks

### DISCUSSION THREAD

**Discussion by Dhawal-Goyal** [pybamm-team/PyBaMM#2756] [Q&A] [answered]
I would like to cycle cell through 1000 cycles while including LAM, SEI and LI platting sub models using drive cycle based discharging. Given that each cycle takes quite some time to run, I am planning to break the entire experiment into multiple sets i.e. i want to simulate say 10 cycles at a time, save the simulation to a pickle file and then load the same as starting solution for next set of 10 cycles.

Can someone help how do I achieve this? 
solve.save saves the output to a file but how do I load it back for using as a starting solution for next set of cycles?

Thanks

**Comment by brosaplanella** [comment, marked as answer]
The notebook on [simulating long experiments](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/simulating-long-experiments.ipynb) provides a starting point for this.

**Reply by Dhawal-Goyal** [reply]
Thanks, I am referring to that already however I am unsure how to load a saved solution so that i can use as starting condition for next set.

**Reply by brosaplanella** [reply]
You can pickle a solution as explained in [this notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Getting%20Started/Tutorial%206%20-%20Managing%20simulation%20outputs.ipynb) and then use the pickled solution and use it as a starting solution as explained in the long experiments notebook.

**Reply by Dhawal-Goyal** [reply]
Thanks this worked fine for me.
